admin/bin/qserv-data-loader.py

```diff
@@ -257,59 +246,17 @@ def run(self):
         return 0
 
 
-    def _czarDbConnect(self):
-        """
-        Connect to czar mysql server, returns connection or throws exception.
-        """
-        kws = dict(local_infile=1)
-        if self.args.mysqlHost:
-            kws['host'] = self.args.mysqlHost
-        if self.args.user:
-            kws['user'] = self.args.user
-        if self.args.password:
-            kws['passwd'] = self.args.password
-        if self.args.mysqlPort:
-            kws['port'] = self.args.mysqlPort
-        if self.args.mysqlSocket:
-            kws['unix_socket'] = self.args.mysqlSocket
-
-        try:
-            return mysql.Connection(**kws)
-        except Exception as exc:
-            logging.critical('Failed to connect to mysql database: %s', exc)
-            raise
-
-
-    def _workerDbConnect(self, nodeName, qservAdmin):
```

Comment:
I'll rename it to `css`, qserv and admin are not really relevant here. We'll refactor/rename whole QservAdmin thing in the future, I guess.